rubato/utils/vector.py

```diff
@@ -523,6 +536,21 @@ def __truediv__(self, other: any) -> Vector:
     def __rtruediv__(self, other: any) -> Vector:
         return Vector(other / self.x, other / self.y)
 
+    def __itruediv__(self, other: any) -> Vector:
+        return self.__truediv__(other)
+
+    def __floordiv__(self, other) -> Vector:
+        if isinstance(other, (int, float)):
+            return Vector(self.x // other, self.y // other)
+        if isinstance(other, Vector):
+            return Vector(self.x // other.x, self.y // other.y)
+
+    def __rfloordiv__(self, other: any) -> Vector:
+        return Vector(other // self.x, other // self.y)
+
+    def __ifloordiv__(self, other: any) -> Vector:
+        return self.__floordiv__(other)
+
     def __neg__(self) -> Vector:
         return Vector(-self.x, -self.y)
 
```

Comment:
Can you check with [the python docs](https://docs.python.org/3/reference/datamodel.html#emulating-numeric-types) to make sure we cover the biggest operators?

Reply:
Sure.

Reply:
Ok, changes implemented, however, we only cover the operators that make sense for vectors. ie. divmod not included, lft/rt shift as well as bitwise operations.